support-frontend/assets/pages/subscriptions-redemption/subscriptionsRedemptionReducer.ts

```diff
@@ -63,10 +63,6 @@ export type Action =
 			type: 'SET_ERROR';
 			error: Option<string>;
 	  }
-	| {
-			type: 'SET_STAGE';
-			stage: Stage;
-	  }
```

Comment:
was this unused?

Reply:
Yeah, it's a duplicate declaration.

Reply:
@TomPretty the remaining declaration is on line 50

Reply:
nice!